### PR TITLE
Onboarding CLI

### PR DESCRIPTION
**Pull request by samiyaakhtar**

### CODE REVIEW

File: cli/operations.py [hunk 1]
```diff
@@ -54,6 +54,35 @@ def get_azure_storage_client(config):
...
 
 
+def onboard(config, folder_name):
+    blob_storage = get_azure_storage_client(config)
+    uri = 'https://' + config.get("storage_account") + '.blob.core.windows.net/' + config.get("storage_container") + '/'
+    functions_url = config.get('url') + '/api/onboarding'
+    images = []
+    count = 0
+    for image in os.listdir(folder_name):
+        if image.lower().endswith('.png') or image.lower().endswith('jpg') or image.lower().endswith('jpeg'):
```
Comment (andrebriggs) [inline cli/operations.py]: @samiyaakhtar How do we know that we only support these file types? Does VOTT say this? Also why a dot for PNG and not for JPG or JPEG?

**Review by andrebriggs — COMMENTED** [review]
@samiyaakhtar not necessarily required for this pull request but think about how the unit tests [here](https://github.com/CatalystCode/active-learning-detect/blob/master/cli/test_operations.py) will change with this new feature.

File: cli/operations.py (hunk 1 above)
Comment (samiyaakhtar) [reply cli/operations.py]: For now the onboard method is very simple: it just takes everything in the folder and uploads it. So this was just a simple sanity check and not meant to be fully exhaustive, but I can look into what VoTT supports.

File: cli/operations.py [hunk 2]
```diff
@@ -54,6 +54,35 @@ def get_azure_storage_client(config):
...
+            local_path=os.path.join(folder_name, image)
+            blob_name = str(uuid.uuid4())
+            count = count + 1
+            print('Uploading image #' + str(count))
+
+            # Upload the created file, use blob_name for the blob name to prevent clashing
+            blob_storage.create_blob_from_path(config.get("storage_container"), blob_name, local_path, content_settings=ContentSettings(content_type='image/png'))
```
Comment (bnookala) [inline cli/operations.py:71]: this content type may not work properly if the image is anything other than a png

Comment (samiyaakhtar) [reply cli/operations.py:71]: What's the right content type to use? It works with the images I tested with

File: cli/operations.py (hunk 1 above)
Comment (mtarng) [reply cli/operations.py]: https://github.com/Microsoft/VoTT/blob/dev/src/index.js#L460
https://github.com/Microsoft/VoTT/blob/dev/src/index.js#L492

These might be the supported image types, but unknown about video.

Comment (samiyaakhtar) [reply cli/operations.py]: Updated it to support these

File: cli/operations.py (hunk 2 above)
Comment (bnookala) [reply cli/operations.py:71]: oh if that's the case, I'm probably wrong … :)

**Comment by samiyaakhtar** [comment]
Here's a sample uploaded gif image https://saakhtastoragetest.blob.core.windows.net/images/3eb8860e-c576-4955-99cf-f13b20fa749b the content type is preserved

**Review by bnookala — APPROVED** [review]

**Pull request merged**